app/konnect/runtime-manager/plugins.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Plugins
+content_type: explanation
+---
+
+You can extend {{site.konnect_short_name}} by using plugins. Kong provides a set of standard Lua plugins that get bundled with {{site.konnect_short_name}}. The set of plugins you have access to depends on your installation.
+
+## Dynamic plugin ordering
+
+By default, plugins execution order is static. You can override the static priority for any {{site.konnect_short_name}} plugin by using each plugin's
+dynamic plugin ordering settings field. This determines plugin ordering during the `access` phase,
+and lets you create _dynamic_ dependencies between plugins. 
+
+To configure this setting in {{site.konnect_short_name}}, go to **Runtime Manager > Plugins**, and then select **Configure Dynamic Ordering** from the context menu next to the plugin you want to configure. From the plugin ordering settings, you can configure whether a plugin runs before or after another plugin.
+
+## Plugin execution order
```

Comment:
I think this could sit in a reference doc all by itself. That would free you up to make this a proper overview of plugins with a picture and a menu bar describing what actions can be taken from the UI.